### PR TITLE
[Term Entry] SwiftUI ViewModifier: `.bold()`

### LINKED ISSUE #1333: [Term Entry] SwiftUI ViewModifier: `.bold()`
### Reason/inspiration (optional)

Please write a new entry under: content/swiftui/concepts/viewmodifier/terms/ about the modifier `.bold()` in Swift.

- A "Syntax" section with pseudo code snippet.
- An "Example" section with a swift code snippet and image to accompany. Images should go in the media/ folder.

To get started with the PR and entry file:

Under content/swiftui/concepts/viewmodifier/terms/, create a new folder called `bold`. Then create a new file called `bold.md` under content/swiftui/concepts/viewmodifier/terms/`bold`

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by gracekishino**
### Description

* Added new folder `terms` inside `content/swiftui/concepts/viewmodifier/`.
* Added new folder `bold` inside `content/swiftui/concepts/viewmodifier/terms/`.
* Added new term entry file `content/swiftui/concepts/viewmodifier/terms/bold/bold.md` for SwiftUI ViewModifier: `.bold()`. 
* Added new image `media/swiftui-bold.png` for the SwiftUI ViewModifier: `.bold()` term entry.

Closes #1333

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by dakshdeepHERE** [comment]
hey @gracekishino  can you tell me how you passed the lint check?

**Comment by gracekishino** [comment]
> 

@dakshdeepHERE it just passed OK first time… might have some ideas so I’ll reply to you in #1409 😊

File: content/swiftui/concepts/viewmodifier/terms/bold/bold.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+Title: '`.bold()`'
```
Comment (SSwiniarski) [inline content/swiftui/concepts/viewmodifier/terms/bold/bold.md:2]:
```suggestion
Title: '.bold()'
```

File: content/swiftui/concepts/viewmodifier/terms/bold/bold.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+Description: 'Is a `ViewModifier` that applies a bold style to text characters in a `View`.'
```
Comment (SSwiniarski) [inline content/swiftui/concepts/viewmodifier/terms/bold/bold.md:3]:
```suggestion
Description: 'A ViewModifier that applies a bold style to text characters in a View.'
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@gracekishino I did a first review and left a couple of comments.

**Comment by gracekishino** [comment]
@SSwiniarski thank you for reviewing so quickly! I committed your suggestions.

**Review by SSwiniarski — APPROVED** [review]
@gracekishino This looks good for a second review

**Comment by Dusch4593** [comment]
Hi @gracekishino and happy New Year! I'll perform the next review 😄

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@gracekishino Just had a few comments/suggestions 😄

**Comment by gracekishino** [comment]
> @gracekishino Just had a few comments/suggestions 😄

@Dusch4593 thanks so much for reviewing and for the comments/suggestions. I committed the suggestions, hope it looks OK now :)

**Review by Dusch4593 — APPROVED** [review]
@gracekishino LGTM for merge! 👍🏻

**Comment by Dusch4593** [comment]
Thank you for another wonderful Docs contribution, @gracekishino! 🚀 🎊 🙌🏻

The entry should now be live: https://www.codecademy.com/resources/docs/swiftui/viewmodifier/bold

Screen Shot 2023-01-05 at 7 35 42 AM

Looking forward to your next Docs contribution! 😄

**Comment by lucwhite** [comment]
Hello @gracekishino,

I hope this message finds you well! My name is Lucas, and I work on the Curriculum team at Codecademy. Our team is considering ways to encourage more individuals to contribute to Codecademy Docs and improve the contribution experience. I was wondering if you would be willing to participate in an interview to provide information on your background and experience with Docs contribution?

The interview would take about an hour and be conducted over Zoom. Should you agree to participate, we will find a time window using Calendly that works for you within the next three weeks. Let us know if you are interested in participating in an interview or have any questions by responding with your email address or emailing me at the address below!

Best,
Lucas White
lucas@codecademy.com

**Pull request merged**